go.mod

```diff
@@ -1,44 +1,78 @@
 module github.com/Orange-OpenSource/casskop
 
-go 1.12
+go 1.13
+
+require github.com/Azure/go-autorest v12.2.0+incompatible
```

Comment:
why is that require not with its other friends below ?

Reply:
Because I had a doubt about it. I required it to fix the following issue: 

```log
build github.com/Orange-OpenSource/casskop/cmd/manager: cannot load github.com/Azure/go-autorest/autorest: ambiguous import: found github.com/Azure/go-autorest/autorest in multiple modules:
	github.com/Azure/go-autorest v10.15.5+incompatible (/Users/aguitton/go/pkg/mod/github.com/!azure/go-autorest@v10.15.5+incompatible/autorest)
	github.com/Azure/go-autorest/autorest v0.9.0 (/Users/aguitton/go/pkg/mod/github.com/!azure/go-autorest/autorest@v0.9.0)
```

Based on this comment : https://github.com/kubernetes/client-go/issues/628#issuecomment-508487691